app/config/AppConfig.scala

```diff
@@ -40,6 +40,9 @@ trait AppConfig {
 
   def minimumTaxV2Foreign: Int
   def minimumTaxV2Uk: Int
+
+  def minimumTaxHistoricFHL: Int
+  def maximumTaxHistoricFHL: Int
```

Comment:
FHL and non-FHL have the same min and max (and the application config parameter name does not make the distinction anyway): so maybe rename  `minimumTaxHistoric`/`maximumTaxHistoric`